js-exercises/bouncer/bouncer.js

```diff
@@ -0,0 +1,8 @@
+function bouncer(array) {
+  let arrayContainingTruthy =  array.filter(element =>{
+        return element
+    })
```

Comment:
A good program overall. I learned from this code, as I didn't know just filtering would work without any condition.
Thanks :100: